csrc/scheduler/registry.h

```diff
@@ -69,6 +69,18 @@ class SchedulerRuntimeInfo : public NonCopyable {
   //!  return max_alignment_size_in_byte.
   size_t getAlignmentSize(TensorView* tv);
 
+  //! Returns strides of tensor, in elements instead of bytes. Only works for
+  //! complete Fusion inputs.
+  const std::vector<int64_t>& getInputStrides(TensorView* tv) {
```

Comment:
Can we return the stride in the language of allocation domain instead of rfactor domain?

Reply:
Yeah I can do that. Essentially it's just a reverse sort but we can do that when saving it instead of at the call site.